src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -677,10 +677,17 @@ private static boolean isWithinIPv4OctetRange(int number) {
 
     public static Map<String, Set<String>> getComputerTags(Computer computer) {
         Set<LabelAtom> labels = null;
-        try {
-            labels = computer.getNode().getAssignedLabels();
-        } catch (NullPointerException e) {
-            logger.fine("Could not retrieve labels");
+        assert computer != null : "Computer is unexpectedly null";
+        var node = computer.getNode();
```

Comment:
I understand that style is subjective, but I think the usage of `var` makes the code less readable compared to specifying the type explicitly

Reply:
Changed.